src/ethereum/frontier/vm/instructions/stack.py

```diff
@@ -81,6 +81,8 @@ def swap_n(evm: Evm, item_number: int) -> None:
     """
     Swap the 1st and Nth items in the stack. All items are 0-indexed from the
     top of the stack.
+    If N is 0, then this is a Noop. Regardless, `swap_n` with N as 0 isn't
+    a valid EVM opcode and hence shouldn't be used.
```

Comment:
I just wanted to clarify this a bit better, since I was confused between `1st` and `All items are 0-indexed`. Hope that's okay!

```suggestion
    Swap the top and the `item_number` element of the stack, where
    the top of the stack is position zero. If `item_number` is zero, this
    function does nothing (which should not be possible, since there
    is no `SWAP0` instruction.)
```

Reply:
Makes sense. These changes would be reflecting in #248